invenio_app_rdm/config.py

```diff
@@ -468,7 +468,7 @@ def files_rest_permission_factory(obj, action):
                        "UIJSONSerializer")
     },
     "datacite": {
```

Comment:
we should also change the key in the dictionary, no?
```suggestion
    "datacite-json": {
```